packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -83,41 +83,45 @@ storiesOf('Button', module)
   )
   .add(
     'element: a, href: \'//example.com\', target: \'_self\'',
-    () => render({elementType: 'a', href: '//example.com', target: '_self', variant: 'primary'})
+    () => render({elementType: 'a', href: '//example.com', target: '_self', variant: 'primary'}, 'Note: Known browser behavior where keyboard focus in Safari doesn\'t work.')
```

Comment:
lets use the storybook description capabilities https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/tabs/stories/Tabs.stories.js#L27